docs/sources/enterprise/access-control/usage-scenarios.md

```diff
@@ -132,6 +132,56 @@ Example response:
 }
 ```
 
+## Manage roles granted directly to users
+
+There are two ways to grant roles directly to users: in the UI using the role picker, and using the API.
+
+### Grant roles to a user within a specific Organization using the role picker
+
+In order to assign roles to a user within a specific Organization using the role picker, you must have a user account with one of the following: 
+* The Admin builtin role.
+* The Server Admin role.
+* The fixed role `fixed:permissions:writer`, [assigned for the given Organization]({{< relref "roles/#scope-of-assignments" >}}).
+<!-- - A custom role with `[INSERT PERMISSIONS HERE]` permissions -->
```

Comment:
@vtorosyan, can someone have a look at this line, as well as lines 162 and 181?

Reply:
Alternatively, we can remove these embedded TODOs for now and track them as a new issue.

Reply:
> We need to merge changes from https://github.com/grafana/grafana/pull/42159/files which might be better/relevant.

@IevaVasiljeva has restarted the build.